packages/entities/src/lib/update.mutation.ts

```diff
@@ -47,20 +48,27 @@ export function updateEntities<
   updater: U,
   options: BaseEntityOptions<Ref> = {}
 ): Reducer<S> {
-  return function (state) {
+  return function (state, context, action) {
     const { ref: { entitiesKey } = defaultEntitiesRef } = options;
     const updatedEntities = {} as Record<
       getIdType<S, Ref>,
       getEntityType<S, Ref>
     >;
 
-    for (const id of coerceArray(ids)) {
+    const coerceIds = coerceArray(ids);
+    for (const id of coerceIds) {
       updatedEntities[id] = toModel<getEntityType<S, Ref>>(
         updater,
         state[entitiesKey][id]
       );
     }
 
+    // TODO @NetanelBasal
+    // Should we check if entity exist and have been realy update
+    if (coerceIds.length) {
```

Comment:
you can move this check to the top and return the original state if the ids array is empty